### DISCUSSION TITLE
Problems upgrading my workspace plan

### DISCUSSION THREAD

**Discussion by ricardocaste** [AntlerVC/firetable#1295] [Support / Q&A] [answered]
I want to upgrade my workspace plan. 
After add card information and click on the update button, appears a popup with the text: "Welcome to Rowy Pro! 🎉
Enabling pro feature" that never ends. 
It has already been 5 hours.

**Comment by BeeBombshell** [comment, marked as answer]
Hey @ricardocaste, sorry to hear about this... Could you share your Email and Project ID so that we can check the status on our side and help you with this?
CC: @bofeiw

**Reply by BeeBombshell** [reply]
Closing this discussion as inactive... If the issue still persists, feel free to create a new discussion.
